lib/eldap/src/eldap.erl

```diff
@@ -234,6 +271,12 @@ Modify the password of a user.
   password. If not required, use `modify_password/3`.
 """.
 -doc(#{since => <<"OTP 18.0">>}).
+-spec modify_password(Handle, Dn, NewPasswd, OldPasswd) -> ok | {error, term()} | {ok, GenPasswd} when
```

Comment:
According to the old doc this function can also return "{ok, {referral, referrals()}}".

Reply:
fixed